packages/framework/src/Model/Product/Parameter/ParameterGroupFacade.php

```diff
@@ -118,6 +129,9 @@ public function deleteById($parameterGroupId)
 
         $this->dispatchParameterGroupEvent($parameterGroup, ParameterGroupEvent::DELETE);
 
+        $affectedProductIds = $this->productFacade->getIdsByParameterGroupId($parameterGroupId);
+        $this->productRecalculationDispatcher->dispatchProductIds($affectedProductIds, ProductRecalculationPriorityEnum::REGULAR, [ProductExportScopeConfig::SCOPE_PARAMETERS]);
```

Comment:
Can you move the whole exporting to the `DispatchAffectedProductsSubscriber`? 
So in the create/edit/delete you just dispatch the parameter group event (like it's on line 130), and the heavy lifting will do the subscriber.